### DISCUSSION TITLE
Create a static binary

### DISCUSSION THREAD

**Discussion by n0madic** [vlang/v#11583] [Ideas]
I would like to be able to create static portable binaries with cross-compilation

**Comment by jeffangelion** [comment]
@medvednikov will compiling with **static** linking be added?

**Comment by medvednikov** [comment]
@jenrus00 yes

**Comment by serge-hulne** [comment]
What is the status of static compilation under MacOS and Windows 11 today (2023/01/07) ?
- is static compiling available?
- What are the compiler options required ?
